src/main/scala/uk/gov/nationalarchives/tdr/api/http/Routes.scala

```diff
@@ -23,6 +25,29 @@ class Routes(val config: Config) extends Cors {
   val ttlSeconds: Int = 10
   val url: String = config.getString("auth.url")
 
+  def unmarshalEntity(entity: ResponseEntity): String = {
+    Unmarshal(entity).to[String].toString
+  }
+
+  def isNon200Status(status: StatusCode): Boolean = {
+    val code = status.intValue.toString
+    !code.startsWith("2")
+  }
+
+  def logging: Directive0 = {
+    def logNon200Response(req: HttpRequest)(res: RouteResult): Unit = {
+      res match {
+        case Complete(resp) =>
+          if (isNon200Status(resp.status)) {
+            val responseEntity = unmarshalEntity(resp.entity)
+            logger.info(s"Non 200 Response: $req\nStatus Code: ${resp.status}\nResponse Entity: $responseEntity")
```

Comment:
Any idea how the new lines get rendered in CloudWatch? CW has an annoying tendency to split up multiline error messages - I'm hoping there's some way of configuring the logging to avoid that, but I haven't dug into the details.

Reply:
No idea, but removed the line breaks.

Reply:
It sounds like making CloudWatch handle multline logs is possible but a bit fiddly. I've created [TDR-468](https://national-archives.atlassian.net/browse/TDR-468) so that we can come back to this, since it affects all the backend apps.